slang-api/src/test/java/com/sonarsource/slang/impl/MatchTreeImplTest.java

```diff
@@ -35,7 +36,8 @@ public void test() {
     TreeMetaData meta = null;
     Tree expression = new IdentifierTreeImpl(null, "x");
     MatchCaseTree case1 = new MatchCaseTreeImpl(null, null, new LiteralTreeImpl(meta, "42"));
-    MatchTree tree = new MatchTreeImpl(meta, expression, Collections.singletonList(case1));
+    Token keywordToken = new TokenImpl(new TextRangeImpl(1,0,1,20), "match", true);
+    MatchTree tree = new MatchTreeImpl(meta, expression, Collections.singletonList(case1), keywordToken);
     assertThat(tree.children()).containsExactly(expression, case1);
     assertThat(tree.expression()).isEqualTo(expression);
     assertThat(tree.cases()).containsExactly(case1);
```

Comment:
just for completeness sake, we could add an assert on keyword in this unit test too